lwtnn.spec

```diff
@@ -17,7 +17,11 @@ cd ../build
 cmake ../%{n}-%{realversion} \
   -G Ninja \
   -DCMAKE_CXX_COMPILER="g++" \
-  -DCMAKE_CXX_FLAGS="-fPIC" \
+  %ifarch x86_64
+  -DCMAKE_CXX_FLAGS="-fPIC -DEIGEN_MAX_ALIGN_BYTES=64 -msse3" \
+  %else
+  -DCMAKE_CXX_FLAGS="-fPIC -DEIGEN_MAX_ALIGN_BYTES=64" \
+  %endif
```

Comment:
Somehow these settings do not end up being used in `lwtnn` compilation (see https://github.com/cms-sw/cmssw/issues/44188#issuecomment-1973588273, and I also confirmed the behavior locally).

@iarspider @aandvalenzuela Would you be able to take a look what needs to be done here in order to make the `-DEIGEN_MAX_ALIGN_BYTES=64` effective?

Reply:
@iarspider @aandvalenzuela Would you be able to take a look?

Reply:
@makortel , this is because lwtnn was overring the CXX_FLAGS. They have fixed this in new versions https://github.com/lwtnn/lwtnn/commit/2647c25a433c706764fadb123a3a517e87bc6892

Reply:
Thanks @smuzaffar! Let's see if the update to 2.14.1 works better.